staff/gerard-celma/mailchimp-template/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>mailchimp-template</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="body">
+    <header class="header">
+        <div class="header__login">
+            <p>Log in</p>
+        </div>
+        <div class="header__body">
+            <img src="img/main_logo.png" alt="mailchimp logo" class="header__logo">
+            <button class="button">Sign Up Free</button>
+                <nav class="header__links">
+                    <a href="#">Pricing</a>
+                    <a href="#">Features</a>
+                    <a href="#">Resources</a>
+                    <a href="#">Support</a>
+                    <a href="#">Blog</a>
+                    <div class="header__search">
+                        <p>Search</p>
+                        <img src="img/search.png" alt="search">
+                    </div>
+                </nav>
+            </div>
+            </div>
+        </div>
+    </header>
+    <main class="main">
+        <aside class="asidetop">
+            <img src="img/easy-email-newsletters-banner-png8.png" alt="momkey" class="asidetop__monkey">
+            <div class="asidetop__signup">
+                <h2>Easy Email</h2>
```

Comment:
There was an image in the files to put that title!

Reply:
Hostia i didn’t saw it, will check